openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -371,3 +370,49 @@ def must_be_positive(cls, v):
                       "intervals must be positive")
             raise ValueError(errmsg)
         return v
+
+
+class SimulationSettingsMD(SimulationSettings):
+    """
+    Settings for simulation control for plain MD simulations, including
+    writing outputs
+    """
+    class Config:
+        arbitrary_types_allowed = True
+
+    equilibration_length_nvt: unit.Quantity
+    """
+    Length of the equilibration phase in the NVT ensemble in units of time. 
+    The total number of steps from this equilibration length
+    (i.e. ``equilibration_length_nvt`` / :class:`IntegratorSettings.timestep`)
+    must be a multiple of the value defined for
+    :class:`IntegratorSettings.n_steps`.
+    """
+    # reporter settings
+    output_filename = 'simulation.xtc'
```

Comment:
```suggestion
    production_trajectory_filename = 'simulation.xtc'
```
Or something like this? I'm mostly wondering if we should be a bit more explicit in our names so that folks can easily understand what each thing does (we've historically not done a great job at this with other settings, but this is a good starting point).